data/frontier jobs.txt

```diff
@@ -0,0 +1,156 @@
+		dialog "Your student thanks you for the able tutelage and for providing a guided view of some of the galaxy's rougher areas. You receive your payment of <payment>."
+	npc
+		government "Pirate"
+		personality nemesis waiting
+		fleet "Small Northern Pirates"
+		system
+			distance 3 7
+	npc accompany save
+		government Merchant
+		personality escort heroic vindictive
+		fleet
+			names "civilian"
+			variant
+				"Sparrow"
+			variant
+				"Star Barge"
+			variant
+				"Shuttle"
+			variant
+				"Heavy Shuttle"
+
+
+
+mission "Small Pirate gambling"
+	description "Survive the pirate attack!"
+	minor
+	repeat
+	to offer
+		random < 3
+		"reputation: Pirates" < 0
+		"combat rating" < 200
+		credits > 14357
+	source
+		attributes "dirt belt" "south" "rim" "pirate"
+	on offer
+		conversation
+			`	You're having a drink in a dimly lit spaceport bar when you notice a couple of young men in crimson leather outfits gambling in a particularly dark corner. The barman eyes them warily, and you suspect they may be pirates. One of them raises his drink and waves you over, beckoning to join their game.`
+			choice
+				`(Join their game.)`
+					goto gamble
+				`(Ignore them and leave quickly.)`
+					defer
+
+			label gamble
+			`	You saunter confidently over to the table. A closer examination all but confirms your suspicion that they're pirates. Not one of them looks older than 19, and they all sport fearsome tattoos and dangerous-looking sidearms. You put on a brave face and they deal you into their card game. After a few rounds it becomes abundantly obvious that they're cheating, and making little effort to hide this from you.`
+			branch "good luck" "bad luck"
+				random < 30
+
+			label "good luck"
+			`	Fortunately, you're able to outsmart them anyway and win the game, winding up with a sizable pile of credit chips. They glower but do not molest you as you quickly seek the safety of a public place. You thank your lucky stars: you're nearly 25,000 credits richer and still in one piece after a face-to-face encounter with pirates.`
+				accept
+
+			label "bad luck"
+			`	You're starting to bleed money, with more than 10,000 credits in losses already.`
+			choice
+				`(Call them out on their cheating.)`
+					goto call
+				`(Say nothing and continue playing.)`
+					goto lose
+
+			label lose
+			`	You grit your teeth and lose hand after hand before the game finally, mercifully ends. Discretion may be the better part of valor, but it's lost you almost 15,000 credits. The pirates laugh at you and gloat over their "winnings" as you slink away, your pride stung.`
+				decline
+
+			label call
+			`	This is intolerable. You start to raise a stink about their blatant cheating.`
+			`	"We're just playin' a game here, see? You'd better shut your big mouth if you know what's good for you," the meanest-looking one growls at you. "We can make this ugly real fast."`
+			`	The four thugs stare at you menacingly and start to reach for their guns.`
```

Comment:
Perhaps change this to something like `You try to confront them on their blatant cheating when one of them yells out, "We're just playin' a game here...`, removing the first sentence and merging it with the dialog.
I'd change `The four thugs` to just `The thugs`, as the number of pirates wasn't mentioned previously.